src/App.js

```diff
@@ -1,23 +1,30 @@
 import { useEffect } from "react";
 import "./App.css";
 import { RoutesPath } from "./Routes";
-import { getVideos } from "./serverCalls/getVideos";
-import {useVideoContext} from "./context/videoContext";
+import { getVideos, getCategories } from "./serverCalls/initialCalls";
+import { useVideoContext } from "./context/videoContext";
 
 function App() {
-  const {dispatch} = useVideoContext();
+  const { dispatch } = useVideoContext();
   useEffect(() => {
```

Comment:
Why not make this API call inside the `videoContext`? 
It seems like there is nothing that is specific to the `App` file and you are using it. So why not make this API call in the context?

Reply:
Made the changes, done